docs/examples/use_cases/tensorflow/efficientdet/pipeline/dali/ops.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2021 Kacper Kluk, Paweł Anikiel, Jagoda Kamińska. All Rights Reserved.
```

Comment:
This file name is really confusing - not that I have a much better alternative at hand, but being inside a directory called `dali` it is a bit too similar to `dali.ops`.

Reply:
I have no good idea for a name too, would one from these: utils, helpers, helper_ops work?

Reply:
I decided to use ops_util.